cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -134,6 +141,7 @@ If run with the "--dry-run" flag, atlas will not execute any SQL.`,
 	cmd.Flags().StringVarP(&flags.txMode, flagTxMode, "", txModeFile, "set transaction mode [none, file, all]")
 	cmd.Flags().BoolVarP(&flags.allowDirty, flagAllowDirty, "", false, "allow start working on a non-clean database")
 	cmd.MarkFlagsMutuallyExclusive(flagLog, flagFormat)
+	migrateApplySetFlags(cmd, &flags)
```

Comment:
I think we can simply inline the logic here. Most of the code already resides in this file.

Reply:
Love this ❤️